nym-vpnd/src/command_interface/listener.rs

```diff
@@ -176,6 +200,49 @@ impl NymVpnd for CommandInterface {
             success: status.is_success(),
         }))
     }
+
+    type ListenToConnectionStatusStream = Pin<
```

Comment:
perhaps use `BoxStream` alias: https://docs.rs/futures/latest/futures/stream/type.BoxStream.html for slightly better readibility